src/main/resources/static/js/alert.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
```

Comment:
I'll try and remove a whole bunch of them. Most we won't need and those that we do, I'll see if I can find them in a web jar.